torcs_msgs/msg/TORCSGlobal.msg

```diff
@@ -0,0 +1,21 @@
+std_msgs/Header header
+# Distance of the car mass center from the surface of the track
+# along the Z axis in meters
+float64 z
+# Distance of the car to the world origin 
+# along the X axis in meters
+float64 x
+# Distance of the car to the world origin 
+# along the Y axis in meters
+float64 y
+# Roll of car
+float64 roll
+# Pitch of car
+float64 pitch
+# Yaw of car
+float64 yaw
+# Speed along x axis of global coordinates
+float64 speedGX
+# Speed along y axis of global coordinates
+float64 speedGY
+
```

Comment:
Please use [geometry_msgs/PoseStamped](http://docs.ros.org/api/geometry_msgs/html/msg/PoseStamped.html)  and [geometry_msgs/TwistStamped](http://docs.ros.org/api/geometry_msgs/html/msg/TwistStamped.html) for global pose and speed respectively instead of introducing a new (redundant) msgs-type.